packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -145,22 +144,24 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
             { resolvedUrl: odspResolvedUrl, docId: odspResolvedUrl.hashedDocumentId },
             odspLogger);
 
+        const storageTokenFetcher = this.toInstrumentedOdspTokenFetcher(
+            odspLogger,
+            odspResolvedUrl,
+            this.getStorageToken,
+            true /* throwOnNullToken */,
+        );
+
+        const webSocketTokenFetcher = this.toInstrumentedOdspTokenFetcher(
+            odspLogger,
+            odspResolvedUrl,
+            this.getWebsocketToken,
+            true /* throwOnNullToken */,
```

Comment:
You are changing the behaviour here whether top throw on null token or not while getting web socket token. Please take a look at my other comment realted to this where I guees we don't want to throw on null token for this.

Reply:
Yes, thanks for catching this! It was copy/paste error.